src/common/services/stream-async-iterable.ts

```diff
@@ -0,0 +1,14 @@
+export async function* streamAsyncIterable<T = unknown>(stream: ReadableStream<T>) {
```

Comment:
‌‌‌It seems like this function isn't used anywhere?Can this file be deleted now?

Reply:
Yes, you can delete it.

Reply:
You can also delete reference of api.js  in Manifest, this script is no longer needed to generate the arkoseToken.